test_runner/src/main/kotlin/ftl/run/platform/android/CreateAndroidTestContext.kt

```diff
@@ -119,13 +155,16 @@ internal fun InstrumentationTestContext.getFlankTestMethods(
             .filter(testFilter.shouldRun)
             .filterNot(parameterizedClasses::belong)
             .map(TestMethod::toFlankTestMethod).toList()
-            .plus(parameterizedClasses.onlyShouldRun(testFilter))
+            .plus(parameterizedClasses.onlyShouldRun(testFilter, parameterizedTests.shouldIgnore()))
```

Comment:
`#shouldIgnore` is a simple method, I think we can return an empty list here instead of inside `onlyShouldRun` method
```suggestion
            .plus(if (parameterizedTests.shouldIgnore()) emptyList() else parameterizedClasses.onlyShouldRun(testFilter))
```

Not saying it have to look like this, but I'm sure you got the idea.
WDYT?

Reply:
Looking at simply neatness i think having an if statement in a chained set of function calls looks worse than separating it out like is being done for all the other chained function calls.

Reply:
Well, it doesn't have to be a chained call. I'm saying that we can avoid passing a boolean parameter that will be used only to decide if we should return an empty list or not. That's all :)